pkg/provider/gcp/secretmanager/secretsmanager_test.go

```diff
@@ -11,7 +11,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package secretmanager
+package secretmanager_test
```

Comment:
I see that we are using https://github.com/maxbrunsfeld/counterfeiter for fake tests, I think we should discuss that addition and decide if we want to use it everywhere.

cc @moolen @gusfcarvalho

Reply:
I think we can discuss this one on the next community meeting :) @BooleanCat do you think you can join it?